addons/mail/static/src/models/rtc_call_viewer/rtc_call_viewer.js

```diff
@@ -183,17 +183,14 @@ registerModel({
          * @private
          */
         _computeMainParticipantCard() {
-            if (!this.messaging || !this.threadView) {
+            if (!this.messaging || !this.focusedRtcSession || !this.threadView || !this.focusedRtcSession) {
                 return unlink();
             }
-            if (this.messaging.focusedRtcSession && this.messaging.focusedRtcSession.channel === this.threadView.thread) {
-                return insert({
-                    relationalId: `rtc_session_${this.messaging.focusedRtcSession.localId}_${this.threadView.thread.localId}`,
-                    rtcSession: link(this.messaging.focusedRtcSession),
-                    channel: link(this.threadView.thread),
-                });
-            }
-            return unlink();
+            return insert({
+                relationalId: `rtc_session_${this.focusedRtcSession.localId}_${this.threadView.thread.localId}_main_card`,
```

Comment:
it was already like this before changes in this PR, but is it possible to just remove `RtcCallParticipantCard/relationalId` and instead use `RtcCallParticipantCard/rtcSession` as identifying field?
This `relationalId` field is very ugly...

Reply:
Currently, it's like that because some cards don't have a rtcSession (but a guest or a partner), I will try with `or` identifying fields with them.

Reply:
I think it works with `[['rtcSession', 'invitedPartner', 'invitedGuest'], ['rtcCallViewerOfMainCard', 'rtcCallViewerOfTile']]` as intenfying fields,